pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```

Comment:
what is this? Have you updated you local pnpm via corepack?

Reply:
Oops, this was not intentional, I will revert this and the settings.json :)